pypsa/networkclustering.py

```diff
@@ -147,12 +147,20 @@ def aggregatebuses(network, busmap, custom_strategies=dict()):
 
 def aggregatelines(network, buses, interlines, line_length_factor=1.0):
 
+    # Make sure network component contents start with "Input" for r and x to avoid missing data in the clustered network before call aggregatelines.
+    network.components["Line"]["attrs"].at['x','status']='Input (required)'
+    network.components["Line"]["attrs"].at['r','status']='Input (required)'
+  
```

Comment:
could you point out why this is necessary?

Reply:
It looks like in the network, line component attribute text for r and x do not start with "Input", so line 165 "columns = set(attrs.index[attrs.static & attrs.status.str.startswith('Input')]).difference(('name', 'bus0', 'bus1'))" could not select r and x? Maybe this is already been fixed in newer version of Pypsa? I guess this may not be the best location to fix here anyways...

Reply:
ah wow, I didn't even know this was used here. Okay, but it actually seems that on the current master it is there.